v2/creating-a-promotion.rst

```diff
@@ -1,4 +1,56 @@
 Create a promotion
 ==================
 
-@todo
+Imagine that your store is currently running a special New Year campaign where you'd like to give customers a 10% discount on orders worth $50 or more. With Commerce Core, it is quite simple.
```

Comment:
> With Commerce Core, it is quite simple.

"With the default offers and conditions provided by the promotion system, it is quite simple."